content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -148,17 +150,19 @@ gcloud.compute.zone | The zone to use for zonal resources; must be in gcloud.com
 
   * Choose the management cluster context
     ```bash
```

Comment:
```suggestion
  * Choose the management cluster context
  
    ```bash
```

Reply:
I'd prefer we tackle this problem by formalizing a yaml formatter and add presubmit check, because it's too costly for everyone to stick to this standard.

Reply:
How would that work? Would it automatically update the Markdown file or would it highlight this as a potential error when testing a PR?

In my opinion, this should not be an automatic update. Stylistically, it is a good practice to add lines between text content and code areas. (Above and below.) It helps with readability of the Markdown and generally seems to help the page render correctly. That said, I've seen some cases where Goldmark hasn't rendered the way I would have expected it to. This has led me to focus on Markdown formatting in PRs when the page renders incorrectly.

Reply:
Agree with that, I believe I've seen cases where trailing spaces (for the indentation) on the empty lines affect formatting. That was really confusing for me.

Additionally, just realized another source of chaos, I've been testing with an outdated hugo binary v0.5x, so I've seen quite a few times my local rendering is different from in the kubeflow.org website. So I was never confident if the doc would render correctly, until it shows up in the preview.
Luckily I just found this, so I fixed it.

In general, I did some quick research, it seems the extra rules like "always adding empty lines before and after code blocks" are not implemented by markdown formatters. However, they can be enforced by linters like https://github.com/DavidAnson/vscode-markdownlint#configure. Install that plugin on your vscode and it automatically highlights problems. Additionally, we could set up the same process in presubmit tests, so it's automatically checked for everyone. I think that's a much better way to get everyone writing markdowns with the same set of best practices. It should also save all the manual review efforts we spend on these.

Reply:
Created https://github.com/kubeflow/website/issues/2381, let's follow up there.